gradle/publish.gradle

```diff
@@ -0,0 +1,67 @@
+apply plugin: 'java'
+apply plugin: 'maven-publish'
+apply plugin: 'com.mapbox.sdkRegistry'
+apply from: file("${rootDir}/gradle/artifact-settings.gradle")
+
+afterEvaluate {
+    publishing {
+        publications {
+            release(MavenPublication) {
+//                from components.release TODO do we need this? it is not found
```

Comment:
I believe this is for Android artifacts. This is a Java project, so it might not be available.